packages/cli/commands/create.js

```diff
@@ -260,6 +260,9 @@ exports.handler = async options => {
         }
       );
       if (created) {
+        if (fs.existsSync(`${filePath}/.env.template`)) {
+          fs.copySync(`${filePath}/.env.template`, `${filePath}/.env`);
+        }
```

Comment:
One thing that I am not sure I understand is why the boilerplates should not be adjusted to have `.env`.

Reply:
Hi, @gcorne, adjusting `.env` to the boilerplates is a good idea, but we have some concerns about excluding it from `.gitignore` since:
- it needs to be gitignored during our development
- user will most likely add it to .gitignore again

So, wondering, how do you see adjusting `.env` to the boilerplates?

Reply:
Alright, I see why using a template was chosen. I think copying from a template is okay, although I wonder if moving the file would be better.